bundlewrap/lock.py

```diff
@@ -24,29 +24,7 @@
 from .utils.ui import io
 
 
-HARD_LOCK_PATH = "/tmp/bundlewrap.lock"
-HARD_LOCK_FILE = HARD_LOCK_PATH + "/info"
-SOFT_LOCK_PATH = "/tmp/bundlewrap.softlock.d"
-SOFT_LOCK_FILE = "/tmp/bundlewrap.softlock.d/{id}"
-
-
-def get_hard_lock_info(node, local_path):
-    try:
-        node.download(HARD_LOCK_FILE, local_path)
-        with open(local_path, 'r') as fp:
-            return json.load(fp)
-    except (RemoteException, ValueError):
-            io.stderr(_(
-                "{x} {node_bold}  corrupted hard lock: "
-                "unable to read or parse lock file contents "
-                "(clear it with `bw run {node} 'rm -Rf {path}'`)"
-            ).format(
-                node_bold=bold(node.name),
-                node=node.name,
-                path=HARD_LOCK_PATH,
-                x=red("!"),
-            ))
-            return {}
+LOCK_BASE = "/var/tmp"
```

Comment:
Hmm, maybe find a better location. It's fine on Linux, but if you want to use an OpenBSD box as a locking node, then it's bad: It clears /var/tmp on boot.

It's surprisingly hard to find a good and portable path. Maybe `/usr/local/var/bw`? Anything below `/usr/local` should be pretty much guaranteed not to be touched by the OS. Then again, OpenBSD installs packages there. Maybe we should even consider using `/root` ...

Reply:
(Is using bw without having root access on a node supposed to work?)